instagram-app/src/components/PostContainer/PostCard.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import User from './User'
+import CommentBox from '../CommentSection/Comments'
+import {FaRegHeart as Heart, FaRegComment as Comment} from 'react-icons/fa'
+import './Posts.scss'
+
+export default class PostCard extends Component {
+    
+    render() {
+        return (
+            <section className="postContainer d-flex flex-column p-1 mx-auto m-5">
+                <User thumb={this.props.postData.thumbnailUrl} user={this.props.postData.username}/>
+                <img 
+                    src={this.props.postData.imageUrl} 
+                    alt={`By ${this.props.postData.username}`}
+                    className='img-fluid mx-auto'
+                />
+                <section className="d-flex flex-column">
+                    <section className="c-icons d-flex flex-row justify-content-start">
+                        <section onClick={()=>this.props.like(this.props.post)}>
+                            <Heart />
+                        </section>
+                        <section onClick={()=>this.props.comment(this.props.post)}>
+                            <Comment/>
+                        </section>

```

Comment:
Great job here. I love that you explicitly wrote out each of your even handlers as anonymous fat arrow functions rather than just the invocation of the relevant function. I know I have mentioned this a few times, this is a best practice and good habit since react can be finicky.